frontend/src/utils/formatting.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * This is thin wrapper on top of the formatter that's passed to
+ * reat-timeago. The desired date is a format that's something like
+ *
+ * "X years and X days"
+ *
+ * which isn't supported by react-timeago out of the box.
+ *
+ * We add a little more custom logic to format calculate what
+ * the number of days is since the input date, and
+ * then append a suffix to pluralize number of days.
+ *
+ * See more on the formatting options, here:
+ * https://github.com/nmn/react-timeago#formatter-optional
+ */
+const yearAndDayFormatter = (value, unit, suffix, epochMiliseconds) => {
+  if (unit === "year") {
+    // The days calculation comes directly from react-timeago:
+    const days = Math.round(Math.abs(epochMiliseconds - Date.now()) / (1000 * 60 * 60 * 24));
```

Comment:
I'm a bit confused about the `epochMiliseconds - Date.now()` part - Shouldn't this result in `0`?
I just checked the react-timeago docs and they describe the fourth argument of the formatter like this:
`epochSeconds: The result of Date.now() or the result of a custom now prop.`

Reply:
Yeah good question! It's non-obvious (I believe the docs for `react-timeago` are incorrect as `Date.now()` returns milliseconds) and took a bit of double checking to work through. I opened a PR into the project to fix it: https://github.com/nmn/react-timeago/pull/151

I'll handle the 0 case, and add a couple more comments outlining it.